camel-core/src/main/java/org/apache/camel/util/EndpointHelper.java

```diff
@@ -363,8 +356,10 @@ public static boolean isReferenceParameter(String parameter) {
                 // The bean is a list
                 return (List) bean;
             } else {
-                // The bean is a list element
-                return Arrays.asList(elementType.cast(bean));
+            	// The bean is a list element
```

Comment:
i am not sure why this is happening. I guess something is wrong with my checkstyle plugin on eclipse. cs fix will be coming soon. sorry about that.